cypress/integration/userCanViewProducts.feacture.js

```diff
@@ -2,7 +2,7 @@ beforeEach(() => {
   cy.intercept("GET", "**/api/products", {
     fixture: "products.json",
   }).as("getProducts");
-  cy.visit("/");
+  cy.visit("/products");
```

Comment:
I made this change to resolve the merge conflict, but the team needs to decide how you want to deal with splitting up articles and products